libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -231,6 +250,7 @@ SRV_Channels::copy_radio_in_out(SRV_Channel::Aux_servo_function_t function, bool
             if (do_input_output) {
                 rc->read();
             }
+            set_output_unlimited_once(function);
```

Comment:
It makes some sense in the failsafe lib, but I don't think it makes sense in ```Plane::set_servos_manual_passthrough``` for example. I'd say that we should be forced to explicitly open the range if needed. ```Plane::set_servos_manual_passthrough``` definetly should be constrained though.